src/main/java/io/camunda/testing/assertions/ProcessInstanceAssertions.java

```diff
@@ -4,6 +4,7 @@
 
 import io.camunda.testing.filters.StreamFilter;
 import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.client.impl.ZeebeObjectMapper;
```

Comment:
:x: Nothing here, but please reword typo in commit message

Reply:
Looks like some dutch slipped through 😄 
I've edited the message